src/lib/components/Swap/Input.tsx

```diff
@@ -1,10 +1,13 @@
 import { Trans } from '@lingui/macro'
+import { LoadingOpacityContainer } from 'components/Loader/styled'
```

Comment:
If this is a simple component, recreate it in `lib`, or move it to `lib`. Right now `lib` does not rely on _any_ components from the app, and adding one in will make it harder to decouple later on.

Reply:
If it is CSS, you can also just import the CSS and target it to child components based on their ReactNode. This would allow you to style the TokenInput without modifying TokenInput - it's also the strategy used for unapproved tokens.

Reply:
Because its component decided to move into Loading component within lib